__tests__/screens/send-bitcoin-screen.tsx

```diff
@@ -0,0 +1,352 @@
+import * as React from "react"
+import { MockedProvider } from "@apollo/client/testing"
+import { gql, InMemoryCache } from "@apollo/client"
+import { act, cleanup, fireEvent, render, waitFor } from "@testing-library/react-native"
+import moment from "moment"
+import "@testing-library/jest-native/extend-expect"
+import "../../node_modules/react-native-gesture-handler/jestSetup.js"
+
+import { translate } from "../../app/i18n/translate"
+import "../../__mocks__/react-native-firebase"
+import {
+  LIGHTNING_PAY,
+  PAY_KEYSEND_USERNAME,
+  SendBitcoinScreen,
+} from "../../app/screens/send-bitcoin-screen"
```

Comment:
we can use moduleNameMapper to avoid this :)

https://github.com/GaloyMoney/galoy/blob/main/jest.config.js#L12